build/update.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Update Nextcloud server and apps from latest git master
+# For local development environment
+# Use from Nextcloud server folder with `./build/update.sh`
+
+# Update server
+printf "\n\033[1m${PWD##*/}\033[0m\n"
+git checkout master
+git pull --quiet
```

Comment:
Could we make it like this to also remove remote branches that are deleted on GitHub?

```suggestion
git pull --quiet -p
```

Reply:
Isn’t that done by the `git fetch --prune --quiet` further down?

Reply:
> Isn’t that done by the `git fetch --prune --quiet` further down?

Yes it is, but you could do this in one go, by doing it in the `pull`